pyvelox/pyvelox.h

```diff
@@ -109,6 +111,27 @@ inline velox::variant pyToVariant(const py::handle& obj, const Type& dtype) {
   }
 }
 
+inline void checkRowVectorBounds(const RowVectorPtr& v, vector_size_t idx) {
+  if (idx < 0 || size_t(idx) >= v->childrenSize()) {
+    throw std::out_of_range("Index out of range");
+  }
+}
+
+inline VectorPtr getVectorFromRowVectorPtr(
+    const RowVectorPtr& v,
```

Comment:
nit: having short names for vector (like v) goes against convention in this file and in velox.